src/ast.ts

```diff
@@ -174,6 +176,15 @@ export interface SequenceExpression extends BaseExpression {
   expressions: Expression[]
 }
 
+export interface ListExpression extends BaseExpression {
+  type: 'ListExpression'
+  elements: Expression[]
+}
+
+export interface EmptyListExpression extends BaseExpression {
```

Comment:
Should we just default to using ListExpression type for empty lists too? Since empty list is a valid list. 

(Grammar wise it is ok to separately represent it, since empty list has different syntax forms, but after it gets parsed, shud they all just be treated as lists?)

Reply:
I see! Makes sense!